File-Formats/PDF/Convert-HTML-To-PDF/aws.md

```diff
@@ -7,132 +7,127 @@ documentation: UG
+## Setting up the AWS Toolkit for Visual Studio
 
-## Refer to the following steps to convert HTML to PDF in AWS Lambda:
+* You can create an AWS account by referring to this [link.](https://aws.amazon.com/) 
+* Download and install the AWS Toolkit for Visual Studio, you can download the AWS toolkit from this [link](https://aws.amazon.com/visualstudio/).
+* The Toolkit can be installed from Tools/Extension and updates options in Visual Studio. 
 
-Create an AWS Lambda function to convert HTML to PDF and publish it to AWS.
-Invoke the AWS Lambda function in your main application using AWS SDKs.
+## Refer to the following steps to convert HTML to PDF in AWS Lambda
 
-## Steps to convert HTML to PDF in AWS Lambda:
+* Create an AWS Lambda function to convert HTML to PDF and publish it to AWS.
+* Invoke the AWS Lambda function in your main application using AWS SDKs.
 
-Create a new AWS Lambda project as follows:
+## Steps to convert HTML to PDF in AWS Lambda
+
+1. Create a new AWS Lambda project as follows:
 ![Convert HTMLToPDF AWS Step1](htmlconversion_images/AWS1.png) 
  
-Create a project name and select location.
+2. Create a project name and select location.
 ![Convert HTMLToPDF AWS Step2](htmlconversion_images/AWS2.png) 
 
-Select Blueprint as Empty Function and click Finish.
+3. Select Blueprint as Empty Function and click Finish.
 ![Convert HTMLToPDF AWS Step3](htmlconversion_images/AWS3.png) 
 
-Install the [Syncfusion.HtmlToPdfConverter.Net.Aws](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Aws/) NuGet package as a reference to your AWS lambda project from [NuGet.org.](https://www.nuget.org/)
+4. Install the [Syncfusion.HtmlToPdfConverter.Net.Aws](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Aws/) NuGet package as a reference to your AWS lambda project from [NuGet.org.](https://www.nuget.org/)
 ![Convert HTMLToPDF AWS Step4](htmlconversion_images/AWS4.png) 
 
-Using the following namespaces in the Function.cs file.
-
-{% highlight c# tabtitle="C#" %}
-
-using Syncfusion.HtmlConverter;
-using Syncfusion.Pdf;
-using System.IO;
-
-{% endhighlight %}
-
-Add the following code snippet in Function.cs to create a PDF document.
+5. Using the following namespaces in the Function.cs file.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-//Initialize HTML to PDF converter with Blink rendering engine.
-HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+   using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using System.IO;
 
-//Convert URL to PDF.
-PdfDocument document = htmlConverter.Convert(input);
+   {% endhighlight %}
 
-//Save the document into stream.
-MemoryStream memoryStream = new MemoryStream();
+6. Add the following code snippet in Function.cs to create a PDF document.
```

Comment:
Add the following code snippet in Function.cs to convert HTML to PDF document.

Reply:
Changed the content